app/Exceptions/Handler.php

```diff
@@ -2,75 +2,29 @@
 
 namespace App\Exceptions;
 
-use Exception;
 use Illuminate\Foundation\Exceptions\Handler as ExceptionHandler;
-use Illuminate\Http\Request;
+use Throwable;
 
 class Handler extends ExceptionHandler
 {
     /**
-     * A list of the exception types that are not reported.
+     * The list of the inputs that are never flashed to the session on validation exceptions.
      *
-     * @var array
-     */
-    protected $dontReport = [
-        //
-    ];
-
-    /**
-     * A list of the inputs that are never flashed for validation exceptions.
-     *
-     * @var array
+     * @var array<int, string>
      */
     protected $dontFlash = [
+        'current_password',
         'password',
         'password_confirmation',
     ];
 
     /**
-     * Report or log an exception.
-     *
-     * This is a great spot to send exceptions to Sentry, Bugsnag, etc.
-     *
-     * @param  \Exception  $exception
-     * @return void
+     * Register the exception handling callbacks for the application.
      */
-    public function report(Exception $exception)
+    public function register(): void
     {
-        parent::report($exception);
-    }
-
-    /**
-     * Render an exception into an HTTP response.
-     *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \Exception  $exception
-     * @return \Illuminate\Http\Response
-     */
-    public function render($request, Exception $exception)
-    {
-        if($this->isHttpException($exception))
-        {
-            switch ($exception->getStatusCode())
-            {
-                // not found
-                case 404:
-                    return redirect(route('home'));
-                    break;
-
-                // internal error
-                case '500':
-                    return redirect(route('home'));
-                    break;
-
-                default:
-                    return $this->renderHttpException($exception);
-                    break;
-            }
-        }
-        else
-        {
-            return parent::render($request, $exception);
-        }
+        $this->reportable(function (Throwable $e) {
```

Comment:
This was updated from Laravel itself. See commit here: 
https://github.com/laravel/laravel/commit/4931af14006610bf8fd1f860cea1117c68133e94